src/components/EntitlementsPanelForm/EntitlementsPanelForm.tsx

```diff
@@ -49,15 +49,26 @@ const EntitlementsPanelForm = ({
   removeEntitlements,
   setRemoveEntitlements,
 }: Props) => {
+  const {
+    data,
+    isFetching,
+    error: getEntitlementsError,
+  } = useGetEntitlements();
```

Comment:
We'll have to investigate whether the API will give us all of the results if we don't provide a `page` size otherwise we'll be limited to only one page of results.